src/main/java/me/cookiemonster/zsocraft/zsocraftpartyhomes/listener/TabCompleteListener.java

```diff
@@ -18,11 +17,15 @@
 
     private final HashMap<String, Validator> commands_validators = new HashMap<>();
```

Comment:
`private final HashMap<String, Validator> commands_validators = new HashMap<>();`
Ta linijka łamie konwencje nazywania zmiennych (camelCase)

Reply:
nie zwróciłem na to uwagi, już poprawiam